uproot/source/futures.py

```diff
@@ -288,6 +292,7 @@ def _run(self, resource):
         if self._notify is not None:
             self._notify()
             del self._notify
+            self._notify = None
```

Comment:
Adding these should mean that `del blah` is redundant. I guess you want to guarantee the attribute exists even though it may be null?

Reply:
The `del blah` is now redundant, but I just wanted to be 100% sure. I don't think `del` does any magic other than what reassignment does, but whatever.

Anyway, keeping the `del blah` in there expresses the intention more clearly.

Reply:
Yeah as far as I know `del self.blah` is syntactic sugar for `self.__dict__.__delitem__("blah")`.